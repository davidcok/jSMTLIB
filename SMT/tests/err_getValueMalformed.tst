; malformed get-value commands
(set-option :produce-models true)
(set-logic QF_UF)
(declare-fun x () Bool)
(assert true)
(check-sat)
(get-value)
(get-value x)
(get-value x y)
(get-value (x) ) ; OK
(get-value (x) x)
(get-value () )
