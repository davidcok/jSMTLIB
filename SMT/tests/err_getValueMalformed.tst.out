unsupported
success
success
success
sat
(get-value)
          ^
(error "Expected a parenthesized list of terms beginning here")
(get-value x)
           ^
(error "Expected a parenthesized list of terms beginning here")
(get-value x y)
           ^
(error "Expected a parenthesized list of terms beginning here")
(get-value (x) ) ; OK
^^^^^^^^^^^^^^^^
(error "The get-value command is only valid if :produce-models has been enabled")
(get-value (x) x)
               ^
(error "Too many arguments or extraneous material after the command or missing right parenthesis")
(get-value () )
           ^^
(error "Expected a parenthesized list of at least one term")
